src/components/Settings/ShareOnSocialMedia.js

```diff
@@ -27,7 +27,7 @@ const commonIconStyle = css`
 `;
 
 const ShareIconContainer = styled.div`
-  display: flex;
+  display: flow-root;
```

Comment:
Use flex-wrap, `flow-root` is not supported by all browsers.

Reply:
Done